BEACON-V2-draft4-Model/biosamples/defaultSchema.json

```diff
@@ -5,79 +5,164 @@
+        "id" : "NCIT:C3778",
+        "label" : "Serous Cystadenocarcinoma"
+      }      
+    },
+    "tumorProgression": {
+      "description": "Indicates primary, metastatic, recurrent. RECOMMENDED.",
+      "$ref": "https://raw.githubusercontent.com/ga4gh-beacon/beacon-framework-v2/main/common/ontologyTerm.json",
+      "example": {
+        "id" : "NCIT:C4813",
+        "label" : "Recurrent Malignant Neoplasm"
+      }      
     },
-    "cancerFeatures": {
-      "$ref": "#/definitions/CancerFeature"
+    "tumorGrade": {
+      "description": "Term representing the tumor grade. Child term of NCIT:C28076 (Disease Grade Qualifier) or equivalent.",
+      "$ref": "https://raw.githubusercontent.com/ga4gh-beacon/beacon-framework-v2/main/common/ontologyTerm.json",
+      "example": {
+        "id" : "NCIT:C28080",
+        "label" : "Grade 3a"
+      }      
+    },
+    "pathologicalStage": {
+      "description": "Pathological stage, if applicable, preferably as subclass of NCIT:C28108 - Disease Stage Qualifier. RECOMMENDED.",
+      "$ref": "https://raw.githubusercontent.com/ga4gh-beacon/beacon-framework-v2/main/common/ontologyTerm.json",
+      "example": {
+        "id" : "NCIT:C27977",
+        "label" : "Stage IIIA"
+      }      
+    },
+    "pathologicalTnmFinding": {
+      "description": "Pathological TNM findings, if applicable, preferably as subclass of NCIT:C48698 - Cancer TNM Finding Category. RECOMMENDED.",
+      "type": "list",
+      "items": {
+        "$ref": "https://raw.githubusercontent.com/ga4gh-beacon/beacon-framework-v2/main/common/ontologyTerm.json"
+      }
+    },
+    "diagnosticMarkers": {
+      "description": "Clinically relevant biomarkers. RECOMMENDED.",
+      "type": "list",
+      "items": {
+        "$ref": "https://raw.githubusercontent.com/ga4gh-beacon/beacon-framework-v2/main/common/ontologyTerm.json"
+      }
+    },
+    "procedure": {
+      "description": "The procedure used to extract the biosample. RECOMMENDED.",
+      "$ref": "../common/procedure.json",
+      "example": {
+        "code": {
+          "id": "NCIT:C5189",
+          "label": "Radical Cystoprostatectomy"
+        }
+      }      
+    },
+    "files": {
+      "description": "List of files related to the biosample, e.g. VCF or high-throughput sequencing files.",
+      "type": "list",
+      "items": {
+        "$ref": "../common/file.json"
+      }
+    },
+    "materialSample": {
+      "description": "Status of specimen (tumor tissue, normal control, etc.). RECOMMENDED.",
+      "$ref": "https://raw.githubusercontent.com/ga4gh-beacon/beacon-framework-v2/main/common/ontologyTerm.json",
+      "example": {
+        "id": "EFO:0009656",
+        "label": "neoplastic sample"
+      }      
+    },
+    "sampleProcessing": {
+      "description": "Status of how the specimen was processed.",
+      "$ref": "https://raw.githubusercontent.com/ga4gh-beacon/beacon-framework-v2/main/common/ontologyTerm.json",
+      "example": {
```

Comment:
Is the example repeated from "sampleProcessing"?

Reply:
C&P from "materialSample" ... :-/

Reply:
There was no example in the PXF documentation. I understand what this is getting at & now provide an example, but am also fine w/ having it removed (`sampleProcessing` and `sampleStorage`).